config/scripts/run_letsencrypt.sh

```diff
@@ -1,8 +1,43 @@
 #!/bin/bash
-if (! [ -z "$STAGING" ]) then
-  echo "Using Let's Encrypt Staging environment..."
-  letsencrypt -n --staging --expand --apache --agree-tos --email $WEBMASTER_MAIL "$@"
-else
-  echo "Using Let's Encrypt Productioun environment..."
-  letsencrypt -n --expand --apache --agree-tos --email $WEBMASTER_MAIL "$@"
-fi
\ No newline at end of file
+#
+
```

Comment:
Please check if the former variable `STAGING` is not null or empty. If this is true, please set the variable `STAGE` to the value `staging`. Else set `STAGE` to `"production"`.

By doing this, this won't be a breaking change.